### ISSUE TITLE
Upgrade to catwalks-1.12.2-4.0.44.jar

### ISSUE THREAD

**Issue by dmf444** [scalda/Foolcraft_3#291]
## FEATURE REQUEST
- [ ] I would like to propose an additional mod.
- [ ] I would like to propose a config change.
- [ ] I would like to propose a custom recipe.
- [x] I would like to propose another change.

## FEATURE INFORMATION
Hello, Mod dev of Catwalks 4 here. I've been getting some error reports from FoolsCraft 3 users, mostly due to certain mods (VanillaFix, FoamFix) changing the default rendering system. I wrote a work-around into the newest version of Catwalks 4, which probably should be loaded ASAP.

Basically without this hotfix, most catwalks, will not rnder in the world and appear invisible. 

Sorry if this wasn't the proper way to inform you

**Comment by scalda** [comment]
done for 1.5.2
